src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Mappers/Apprentice/DetailsViewModelMapperTests.cs

```diff
@@ -659,15 +671,17 @@ public class DetailsViewModelMapperFixture
             private DetailsViewModelMapper _sut;
             public DetailsRequest Source { get; }
             public DetailsViewModel Result { get; private set; }
-            public GetApprenticeshipResponse ApiResponse { get; }
-            public GetPriceEpisodesResponse PriceEpisodesApiResponse { get; }
-            public GetApprenticeshipUpdatesResponse GetApprenticeshipUpdatesResponse { get; private set; }
-            public GetDataLocksResponse GetDataLocksResponse { get; private set; }
-            public GetChangeOfPartyRequestsResponse GetChangeOfPartyRequestsResponse { get; private set; }
-            public GetChangeOfEmployerChainResponse GetChangeOfEmployerChainResponse { get; private set; }
+            public GetManageApprenticeshipDetailsResponse ApiResponse { get; }
+            public GetApprenticeshipResponse ApiResponseOld { get; }
+            public GetManageApprenticeshipDetailsResponse.ApprenticeshipDetails ApprenticeshipDetails { get; }
+            public IEnumerable<GetManageApprenticeshipDetailsResponse.PriceEpisode> PriceEpisodes { get; }
+            public IEnumerable<GetManageApprenticeshipDetailsResponse.ApprenticeshipUpdate> ApprenticeshipUpdates { get; private set; }
+            public IEnumerable<GetManageApprenticeshipDetailsResponse.DataLock> DataLocks { get; private set; }
+            public IEnumerable<GetManageApprenticeshipDetailsResponse.ChangeOfPartyRequest> ChangeOfPartyRequests { get; private set; }
+            public IEnumerable<GetManageApprenticeshipDetailsResponse.ChangeOfEmployerLink> ChangeOfEmployerChain { get; private set; }
             public GetNewerTrainingProgrammeVersionsResponse GetNewerTrainingProgrammeVersionsResponse { get; private set; }
             public GetTrainingProgrammeResponse GetTrainingProgrammeByStandardUIdResponse { get; private set; }
-            public GetApprenticeshipDetailsResponse GetApprenticeshipDetailsResponse { get; private set; }
+            //public ApprenticeshipDetails ApprenticeshipDetails{ get; private set; }
```

Comment:
Is that whole line commented? Hard to tell in the onscreen diff, can probably remove it if so

Reply:
removed